axelrod/strategies/rand.py

```diff
@@ -10,3 +12,51 @@ class Random(Player):
 
     def strategy(self, opponent):
         return random.choice(['C', 'D'])
+
+
+class Tullock(Player):
+    """
+    Cooperates for the first 11 rounds then randomly cooperates 10% less often
+    than the opponent has in previous rounds."""
+
+    name = "Tullock"
+    memoryone = False # memory-10
+
+    def strategy(self, opponent):
+        if len(self.history) < 11:
+            return 'C'
+        counter = Counter(opponent.history[-10:])
```

Comment:
Instead of importing the `Counter` (not terribly disruptive to do so but I think this is slightly cleaner):

```
opponent.history[-10:].count('C')
```

Another thing that could be worth doing here is making 10 a parameter?